src/crontab.ts

```diff
@@ -33,6 +34,13 @@ export const startCrontab = async () => {
     onTick: tickWrapper(retrieveUsdPrices),
   }).start();
 
+  // Stores sdex Swaps every minute
+  CronJob.from({
+    cronTime: '*/1 * * * *',
+    onTick: tickWrapper(retrieveSwaps),
+    runOnInit: true,
```

Comment:
leave runOnInit only for jobs that needs to be done before other ones (like populating list of tokens), for this one we don't need it.

```suggestion
```